script/release/ci-release-build.js

```diff
@@ -55,30 +56,80 @@ async function makeRequest (requestOptions, parseResponse) {
   })
 }
 
-async function circleCIcall (buildUrl, targetBranch, job, options) {
+async function circleCIcall (targetBranch, job, options) {
   console.log(`Triggering CircleCI to run build job: ${job} on branch: ${targetBranch} with release flag.`)
   const buildRequest = {
-    'build_parameters': {
-      'CIRCLE_JOB': job
+    'branch': targetBranch,
+    'parameters': {
+      'run-lint': false,
+      'run-build-linux': false,
+      'run-build-mac': false
     }
   }
-
-  if (!options.ghRelease) {
-    buildRequest.build_parameters.UPLOAD_TO_S3 = 1
+  if (options.ghRelease) {
+    buildRequest.parameters['upload-to-s3'] = '0'
+  } else {
+    buildRequest.parameters['upload-to-s3'] = '1'
   }
+  buildRequest.parameters[`run-${job}`] = true
   jobRequestedCount++
-  const circleResponse = await makeRequest({
-    method: 'POST',
-    url: buildUrl,
+  try {
+    const circleResponse = await circleCIRequest(circleCIPipelineURL, 'POST', buildRequest)
+    console.log(`CircleCI release build pipeline ${circleResponse.id} for ${job} triggered.`)
+    const pipelineInfoUrl = `https://circleci.com/api/v2/pipeline/${circleResponse.id}`
+    const workflowId = await getCircleCIWorkflowId(circleResponse.id)
+    if (workflowId === -1) {
+      return
+    }
+    console.log(`CircleCI release build workflow running at https://circleci.com/workflow-run/${workflowId} for ${job}.`)
+    const jobInfoUrl = `https://circleci.com/api/v2/workflow/${workflowId}/jobs`
+    const jobInfo = await circleCIRequest(jobInfoUrl, 'GET')
+    if (!jobInfo.items || jobInfo.items.length !== 1) {
+      console.log('Error retrieving job for workflow, response was:', jobInfo)
+      return
+    }
+    const jobUrl = `https://circleci.com/gh/electron/electron/${jobInfo.items[0].job_number}`
+    console.log(`CircleCI release build request for ${job} successful.  Check ${jobUrl} for status.`)
+  } catch (err) {
+    console.log('Error calling CircleCI: ', err)
+  }
+}
+
+async function getCircleCIWorkflowId (pipelineId) {
+  const pipelineInfoUrl = `https://circleci.com/api/v2/pipeline/${pipelineId}`
+  for (let i = 0; i < 5; i++) {
+    const pipelineInfo = await circleCIRequest(pipelineInfoUrl, 'GET')
+    if (pipelineInfo.workflows.length !== 1) {
```

Comment:
The response has a status property we should check otherwise this could hang for ever

I think the possible values are `pending`, `created` and `error`.  Not 100% sure though

Reply:
it won't hang for ever because we only retry 5 times.  I'll add a check for status though